frontend/bower.json

```diff
@@ -8,14 +8,14 @@
     "bootstrap-sass": "^3.3.6",
     "toastr": "^2.1.2",
     "font-awesome": "~4.5.0",
-    "jquery": "1.11.1",
+    "jquery": ">=1.11.3",
```

Comment:
If I remember right, there's a weird jQuery issue when using versions more recent than 1.11.3. I forget if it's ember or dynamic-forms or what, but something doesn't play nice with >1.11.3 (or 1.13.1? the sentiment stands)